buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -3,6 +3,43 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": true,
```

Comment:
Let's move the two tasks post build. Both tasks should always run, and should continue on failure. I will prepare the changes.